site/content/docs/commands/deploy.en.md

```diff
@@ -40,13 +64,23 @@ If the deployment fails when automatic stack rollback is disabled, you may be re
 rollback of the stack via the AWS console or AWS CLI before the next deployment.
 
 ## Examples
-
 Deploys a service named "frontend" to a "test" environment.
 ```console
- $ copilot deploy --name frontend --env test
+$ copilot deploy --name frontend --env test --deploy-env=false
 ```
 
 Deploys a job named "mailer" with additional resource tags to a "prod" environment.
 ```console
-$ copilot deploy -n mailer -e prod --resource-tags source/revision=bb133e7,deployment/initiator=manual
+$ copilot deploy -n mailer -e prod --resource-tags source/revision=bb133e7,deployment/initiator=manual --deploy-env=false
+```
+
+Initializes and deploys an environment named "test" in us-west-2 under the "default" profile with local manifest,
+then deploys a service named "api"
+```console
+$ copilot deploy --init-env --deploy-env --env test --name api --profile default --region us-west-2
```

Comment:
It occurs to me that `--deploy-env` sort of implies `--init-env`! If a user run `copilot deploy --deploy-env --env test`, where `test` is not initialized yet, perhaps it is totally fair for us to just initialize & deploy `test`, even if `--init-env` isn't specified 💭 

This is non-blocking though! If that makes sense to we all, we probably can do this in a future release!